src/code-components/RichTextArea/RichTextArea.register.ts

```diff
@@ -212,6 +234,18 @@ export function registerRichTextArea(
         valueProp: "value",
         onChangeProp: "onChange",
       },
+      error: {
```

Comment:
I think error state is needless, let's just show error toast if upload failed for some reasons

Reply:
The way I've done it currently is problematic because error will be defined still even if user reuploads the image and it goes correctly then.


How long should the error be defined? One second after it appears? Until next upload? Better not to answer these questions at all.